signature/signature.go

```diff
@@ -199,6 +200,95 @@ func (s untrustedSignature) sign(mech SigningMechanism, keyIdentity string) ([]b
 	return mech.Sign(json, keyIdentity)
 }
 
+// cosignSignature is the payload format for sigstore signing an image using
+// the simple signing format.
+type cosignSignature struct {
```

Comment:
I’m paranoid about misuse — I’d prefer to prefix all the names in this with `untrusted`, compare `untrustedSignature` vs. `Signature`.

---

BTW shouldn’t the internal types like `untrustedSignature` be now renamed to refer to a singing format? We can’t remove the public ones, but at least the internal ones should use clear names. For the public part, we might be able to provide new aliases and deprecate old names.